src/bidiServer/index.ts

```diff
@@ -130,12 +130,12 @@ async function onNewBidiConnectionOpen(
     'about:blank',
   ];
 
-  const executablePath = chromeChannel
-    ? computeSystemExecutablePath({
-        browser: Browser.CHROME,
-        channel: chromeChannel,
-      })
-    : process.env['CHROME_BIN'];
+  const executablePath =
+    process.env['CHROME_BIN'] ??
+    computeSystemExecutablePath({
+      browser: Browser.CHROME,
+      channel: chromeChannel || ChromeReleaseChannel.DEV,
```

Comment:
```suggestion
      channel: chromeChannel ?? ChromeReleaseChannel.DEV,
```

Reply:
Or, even better: Add a default argument to `chromeChannel` in the method signature.